pages/login.tsx

```diff
@@ -20,26 +21,27 @@ type Values = {
   password: string
 }
 
-// TODO: Error Handling for login / signup. Blocked by backend implementation.
-const handleSubmit = async (values: Values) => {
-  const res = await fetch(`${SERVER_URL}/signin`, {
-    method: 'POST',
-    headers: {
-      'content-type': 'application/json'
-    },
-    body: JSON.stringify({
-      username: values.username,
-      password: values.password
+const Login: React.FC<Props> = () => {
+  const [invalidCredentials, setInvalidCredentials] = useState(false)
```

Comment:
Your state logic is to display or not `<Alert />` I would rather name the state:
 `[isAlertVisible setIsAlertVisible] = useState(false)`